go.mod

```diff
@@ -5,95 +5,97 @@ go 1.21
 toolchain go1.21.5
```

Comment:
when I was convinced I was going crazy I tried bumping controller runtime. Didn't hurt so I just left it in this PR